File: Scala/Example12.scala
object Example12 extends App {
    // We can create instance of class
    val s = new Summator
    // Also we can access to it's functions
    s.print
    s.setA(6)
    s.setB(7)
    s.print
    println(s.sum) // = 13
}

// Class Summator
class Summator {
    // fields of class
    var a = 0
    var b = 0

    // Functions of class
    // NOTE: we can concat strings with 
    def print = println("a=" + a + " b=" + b)
    def setA(x : Int) = { a = x }
    def setB(x : Int) = { b = x }
    def sum = a + b
}

File: С#/Example12.cs
using System;

class Example12
{
    static void Main()
    {
        // Create new instance of Class
        Class cl_1 = new Class(10);
        Class cl_2 = new Class(10);
        // Now we can access to instances
        // ...
    }
}

// Our class Class
public class Class
{
    public int k = 0;

    public Class(int k)
    {
        this.k = k;
    }
}